### PR TITLE
Improve Polish translation in help messages.

### PR DESCRIPTION
**Pull request by kacperduras**

### CODE REVIEW

File: src/main/resources/messages/help_pl.yml [hunk 1]
```diff
@@ -36,10 +36,10 @@ section:
 commands:
     authme.register:
         description: 'Rejestracja gracza'
-        detailedDescription: 'Rerejestracja gracza z okreslonym haslem'
+        detailedDescription: 'Rerejestracja gracza z okreslonym hasłem'
```
Comment (ljacqu) [inline src/main/resources/messages/help_pl.yml]: Should `Rerejestracja` be `Rejestracja` or is it really correct?

Comment (kacperduras) [reply src/main/resources/messages/help_pl.yml]: Woops, my fault. Wait a minute ;)

Comment (ljacqu) [reply src/main/resources/messages/help_pl.yml]: Not your fault, just an opportunity to fix it since we have a pull request open :) Thanks

Comment (kacperduras) [reply src/main/resources/messages/help_pl.yml]: Fixed.

**Comment by ljacqu** [comment]
Dziękuję! Although I don't speak a word, Polish holds a special place in my heart just because of how awesome it looks in writing :) It just has so much unique stuff going on! In any case, thanks again, happily merged~

**Pull request merged**